From a0d92616928d46724a3dc6f8c07f38b8e6ec3a26 Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Sat, 5 Dec 2015 10:08:13 +0200
Subject: Guard for building with SOURCE_DATE_EPOCH or POD_MAN_DATE set

The strftime test expects the current time rather than anything
in the environment.

Bug-Debian: https://bugs.debian.org/807086
Patch-Name: fixes/podman-source-date-epoch-testfix.diff
---
 cpan/podlators/t/devise-date.t | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/cpan/podlators/t/devise-date.t b/cpan/podlators/t/devise-date.t
index a6ceba1..8a1c06e 100644
--- a/cpan/podlators/t/devise-date.t
+++ b/cpan/podlators/t/devise-date.t
@@ -14,6 +14,9 @@ use POSIX qw(strftime);
 
 use Test::More tests => 6;
 
+local $ENV{SOURCE_DATE_EPOCH};
+local $ENV{POD_MAN_DATE};
+
 # Check that the results of device_date matches strftime.  There is no input
 # file name, so this will use the current time.
 my $parser = Pod::Man->new;
